Sources/UIDeviceExtensions.swift

```diff
@@ -85,6 +85,18 @@ extension UIDevice {
         return UIDevice.current.userInterfaceIdiom == UIUserInterfaceIdiom.pad
     }
 
+    /// EZSE: Returns path of documents directory
+    class func documentsDirectoryPath() -> String {
```

Comment:
There is a little trick to make sure new files are referenced properly in normal project, example project and tests. People usually miss some part, I should write a checklist about it sometime